scala/private/phases/phase_default_info.bzl

```diff
@@ -0,0 +1,36 @@
+#
+# PHASE: default_info
+#
+# DOCUMENT THIS
+#
+def phase_binary_default_info(ctx, p):
```

Comment:
Could we name these phase_default_info_binary, phase_default_info_library, and phase_default_info_scalatest instead? I think it really improves code navigability when the file name is the prefix for all exported functions.

Reply:
We can. I think that I followed @borkaehw’s pattern and so if we want to change I’d rather send a separate PR with this change for all functions.
I’m not sure which option is better IMHO so I can live with either.
@borkaehw wdyt?
(If I’m mistaken and this is the only place that broke the pattern please LMK)

Reply:
I'm in favor of logically grouped changes, so separate PRs is fine by me. Also this is mostly just nitpicking :).

Reply:
Nitpicking is not a problem for me. Given this codebase is read by so many people over a long time we need things to be clear.
Let’s give @borkaehw some time to respond since I promised him we’ll delay this change after #947

Reply:
Sounds like a reasonable change. I didn't think too much about naming order since we were focused on a lot of other stuffs on that PR. I prefer to have separate PR for naming fix, happy to do so as well.